### PR TITLE
updated node-js-buy example shopify-buy to version ^1.2.0

### PR DESCRIPTION
**Pull request by philspitler**
- updated code to work with newer shopify-buy api
- all tests pass

### CODE REVIEW

File: node-js-buy/views/index.pug [hunk 1]
```diff
@@ -14,23 +14,24 @@ html(lang='en', data-framework='javascript')
...
-                select.Product__option(name=option.name)
-                  each value in option.values
-                    option(value=value.valueOf())
-                      =value
-              label Quantity
-              input.Product__quantity(min='1', type='number', name='quantity', value='1')
-              input(type='hidden' name='checkoutId' value=cart.id)
-              button.Product__buy.button#add-to-cart-button(type='submit')
-                  | Add to cart
+        each collection in collections
```
Comment (swalkinshaw) [inline node-js-buy/views/index.pug]: Is this change needed?

I get an error:

```plain
TypeError: /Users/scott/dev/storefront-api-examples/node-js-buy/views/index.pug:17
    15| 
    16|       .Product-wrapper
  > 17|         each collection in collections
    18|           each product in collection.products
    19|             .Product
    20|               img(src=product.images[0].src)

Cannot read property 'length' of undefined
```

Comment (philspitler) [reply node-js-buy/views/index.pug]: Thank you for pointing this out @swalkinshaw.  This is a mistake on my part.  I am using this as the basis for my own app and missed backing this change out.  I'll fix it and push in the next few minutes.

**Comment by philspitler** [comment]
All cleaned up @swalkinshaw, tests still pass.  Thank you again for pointing that out.

**Comment by swalkinshaw** [comment]
Looks good, thank you
